kclvm/sema/src/resolver/import.rs

```diff
@@ -39,7 +39,7 @@ impl<'ctx> Resolver<'ctx> {
                                     pos: Position {
                                         filename: m.filename.clone(),
                                         line: stmt.line,
-                                        column: Some(1),
+                                        column: None,
```

Comment:
It can be modified in another Issue and PR.

Reply:
fixed